TestRailAutomationTest/Utils/Waits.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+
+namespace TestRailAutomationTest.Utils;
+
+public static class Waits
+{
+    private static readonly TimeSpan DefaultWaitSeconds = TimeSpan.FromSeconds(10);
```

Comment:
According CC blank line is needed here

Reply:
And lets specify Default timeout in config file in order to have ability easily change it